### PR TITLE
Added the new textbox element and updated the page object

### PR DESCRIPTION
**Pull request by tyBouch**

### CODE REVIEW

File: docs/test/textbox.html [hunk 1]
```diff
@@ -5,25 +5,27 @@
...
-		  <input type="checkbox" id="vehicle3" name="vehicle3" value="Boat" checked>
-		  <label for="vehicle3"> I have a boat</label><br><br>
-		  <label for="fname">First name:</label>
-		  <input type="text" id="fname" name="fname" disabled><br><br>
-		  <label for="lname">Last name:</label>
-		  <input type="text" id="lname" name="lname"><br><br>
-		  <input type="submit" value="Submit">
-		  <input type="text" id="hiddenbox" class="hidden">
-		  <input type="text" id="notdisplayedbox" class="notdisplayed">
+			<input type="checkbox" id="vehicle1" name="vehicle1" value="Bike">
```
Comment (pturchinetz) [inline docs/test/textbox.html:14]: Is the tabbing off or is this just a visual thing here?

**Review by pturchinetz — COMMENTED** [review]
looks fine, the formatting seems off though, i could be wrong

File: docs/test/textbox.html (hunk 1 above)
Comment (sampacos) [reply docs/test/textbox.html:14]: looks like it's been changed to be more consistent with the rest of the document. Don't think it's a problem so I'm fine with it.

**Review by sampacos — APPROVED** [review]

File: docs/test/textbox.html (hunk 1 above)
Comment (tyBouch) [reply docs/test/textbox.html:14]: Yeah it wasn't quite tabbed the right way before. Same with the page object which used non standard tabbing before.

**Pull request merged**